SofaKernel/modules/SofaCore/src/sofa/core/Mapping.h

```diff
@@ -145,7 +145,7 @@ class Mapping : public BaseMapping
     /// The displacement is accessed in the parent state using mparams->readDx() .
     /// This method generally corresponds to a symmetric stiffness matrix, but with rotations (which are not a commutative group) it is not the case.
     /// Since some solvers (including the Conjugate Gradient) require symmetric matrices, a flag is set in the MechanicalParams to say if symmetric matrices are required. If so, non-symmetric geometric stiffness should not be applied.
-    void applyDJT(const MechanicalParams* /*mparams = MechanicalParams::defaultInstance()*/ , MultiVecDerivId /*parentForce*/, ConstMultiVecDerivId  /*childForce*/ ) override;
+    void applyDJT(const MechanicalParams* /*mparams = GetDefaultMechanicalParamsInstance()*/ , MultiVecDerivId /*parentForce*/, ConstMultiVecDerivId  /*childForce*/ ) override;
```

Comment:
Should be `sofa::core::mechanicalparams::defaultInstance()` ?

Reply:
For the commented code, I prefer much more to remove them.